src/coreclr/src/gc/gcpriv.h

```diff
@@ -3982,26 +4012,25 @@ class gc_heap
 #endif //SYNCHRONIZATION_STATS
 
 #define NUM_LOH_ALIST (7)
-#define BASE_LOH_ALIST (64*1024)
+    // bucket 0 contains sizes less than 64*1024
+#define BASE_LOH_ALIST_BITS (14)
```

Comment:
this should be 15

Reply:
Right!!!  Because it is the highest bit in `64*1024 - 1`, (zero-based as in `BitScanReverse`). I'll add a comment as well.